Removed repaint on move (useless) + fixed graph drawing areas

dubvulture opened the pull request:
I might be wrong about the "useless" repaint on componentMoved, I'd like to know what led to that decision.

dubvulture committed:
Removed repaint on move (useless) + fixed graph drawing areas

silma85 commented:
Hi, I'm not mantaining the project anymore so edit to your heart's content, I also don't really remember much since it was a fair bit of years ago. Did you check borderline conditions on that "Equality" constraint?
It was my degree thesis project, so any decision could very well be bad design. :)

dubvulture committed:
Update GraphicDialog.java

1) Modifica delle aree (c'era ancora un caso in cui non funzionava)
2) Aggiunta commenti
3) Fix disegno vincolo non esistente

dubvulture committed:
Update GraphicDialog.java

dubvulture commented:
I added some comments explaining how the area to be drawn is selected and modified the "Equality" constraint to be more strict in case of unknown Constants values.
There was also a problem when drawing a non-existent constraint with constant GreaterThan (it was inverted to an empty area even if the inequality stands correct)

silma85 commented:
I'm closing the PR because the project is being modernized to Maven and more modern Java. You are free to resubmit the modifications if you like.
